app/models.py

```diff
@@ -424,6 +439,18 @@ def live_services(self):
     def domain_list(self):
         return [domain.domain for domain in self.domains]
 
+    def get_permissions_list(self):
```

Comment:
If we have the `permissions` backref, we shouldn't need this - we can access `organisation.permissions` directly.

```
sam:~/work/gds/notifications-api [WP-perm-ask-to-join-service]$ flask shell
2023-04-24T09:05:22 app app INFO no-request-id "Logging configured" [in /Users/sam/work/gds/notifications-api/venv/lib/python3.9/site-packages/notifications_utils/logging.py:43]
Python 3.9.15 (main, Dec 22 2022, 19:49:52)
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin
App: app
Instance: /Users/sam/work/gds/notifications-api/instance
>>> from app.models import *
>>> Organisation.query.first().permissions
2023-04-24T09:05:24 app app WARNING no-request-id "Checked out sqlalchemy connection from outside of request/task" [in /Users/sam/work/gds/notifications-api/app/__init__.py:409]
[]
```